tests/serverpod_test_server/test/defaults/server_only/server_only_default_test.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:test/test.dart';
+
+void main() async {
+  group(
+    "Given a class with serverOnly scoped fields with default values",
+    () {
+      test(
+        'when an object is created without specifying serverOnly field, then the default value is used',
+        () {
+          // This test would need a corresponding model in the test protocol
```

Comment:
I think after code-gen you can just make use of the `ServerOnlyDefault` test class here, right?